src/newspage/Entry.js

```diff
@@ -1,23 +1,52 @@
 import React from 'react';
 import styled from 'styled-components';
-import PropTypes from 'prop-types';
 import Title from './Title';
 import Author from './Author';
 import Description from './Description';
 import Activity from './Activity';
 import Activities from './Activities';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 
 const EntryContainer = styled.article`
   background: lightgray;
   border-radius: 3%;
   margin-bottom: 30px;
 `;
 
-export default function Entry({ entry }) {
+const DeleteIcon = styled(FontAwesomeIcon)`
+  font-size: 1rem;
+`;
+
+const EntryHeader = styled.header`
+  background: silver;
+  border-top-right-radius: 5%;
+  border-top-left-radius: 5%;
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  padding: 5px;
+  width: 100%;
+`;
+
+export default function Entry({ entry, onNewsDelete }) {
   const { title, author, activities, description } = entry;
+
+  function onDeleteBtnClick(event, onNewsDelete) {
+    const deletedEntry = {
+      ...entry
+    };
+    onNewsDelete(deletedEntry);
+  }
   return (
     <EntryContainer>
-      <Title>{title}</Title>
+      <EntryHeader>
+        <Title>{title}</Title>
+        <DeleteIcon
+          icon="trash"
+          onClick={event => onDeleteBtnClick(event, onNewsDelete)}
```

Comment:
i think you dont need to handle over the `onNewsDelete` function here. you destructured it in `Entry` so it is accessable for onDeleteBtnClick. so you can write `onClick={() => onDeleteBtnClick()}` and your function only ist `function onDeleteBtnClick() { ... }`